src/components/editor/inputs/InputLang.jsx

```diff
@@ -51,6 +51,10 @@ const InputLang = (props) => {
   }
 
   const handleLangSubmit = (event) => {
+    if (radioButtonValue === "present" && lang === null) {
+      alert("Please select a valid language.")
```

Comment:
No alerts. We should disable the Submit button instead when in bad state.

Reply:
OK, though this may get pretty tricky to detect the bad state (I think you'd need to be constantly checking as the user types in the text box).  May not be worth the trouble if it hasn't come up before.

Reply:
actually came up with an alternate way of doing this (disable submit button and show an inline alert), required some extra state though